ios/docs/Controls/Avatar.md

```diff
@@ -0,0 +1,37 @@
+# Avatar
+
+## Overview
+`Avatar` is a graphical representation of a user, team, or entity. It can display an image, icon, or initials, and can be either circular or square.
+
+The different variations of `Avatar` are pictured below.
+
+#### Circle
+| Image | Initials | Icon | Accent Icon | Outline Icon | Outlined Primary Icon | Overflow |
+| - | - | - | - | - | - | - |
+| ![Avatar-Image.png](.attachments/Avatar-Image.png) | ![Avatar-Initials.png](.attachments/Avatar-Initials.png) | ![Avatar-Icon.png](.attachments/Avatar-Icon.png) | ![Avatar-AccentIcon.png](.attachments/Avatar-AccentIcon.png) | ![Avatar-OutlinedIcon.png](.attachments/Avatar-OutlinedIcon.png) | ![Avatar-OutlinedPrimaryIcon.png](.attachments/Avatar-OutlinedPrimaryIcon.png) | ![Avatar-Overflow.png](.attachments/Avatar-Overflow.png) |
+
+#### Square
+| Image | Initials |
+| - | - |
+| ![Avatar-ImageSquare.png](.attachments/Avatar-ImageSquare.png) | ![Avatar-InitialsSquare.png](.attachments/Avatar-InitialsSquare.png) |
+
+## Usage
+```Swift
```

Comment:
Would it make sense to also add a snippet of the Avatar initialization in SwiftUI?